packages/commands/test/src/bin.ts

```diff
@@ -117,11 +118,13 @@ const runTest = async ({
 
   const testCase: TestCase = {
     beforeTest: async () => {
-      if (!skipRestart) {
+      if (!skipRestart && process.env.PLATFORM === "android") {
```

Comment:
process.env.PLATFORM should be set to "android" by default, otherwise it's a breaking change for Android user

alternatively for now, we can assume that if PLATFORM isn't set it should be android